src/opart_gaussian.c

```diff
@@ -15,21 +31,97 @@ int opart_gaussian(int n_data, double *data_ptr, double penalty, double *cost_pt
 	  return NUM_OF_DATA_VALUES_LESS_THAN_ZERO;
 	}
 
+	int i;
+	int j;
+	int temp;
```

Comment:
please use more informative variable names --
e.g. `change_i` rather than `i` and `candidate_end` rather than `temp`